app/common/balances/BalanceService.ts

```diff
@@ -54,20 +45,14 @@ export const createBalanceService = (connection: Connection): IBalanceService =>
     asset: Asset,
     onUpdate: (result: IAssetBalance) => void,
   ): Promise<() => void> {
-    return connection.api.query.assets.account.multi([[asset.typeExtras?.assetId, address]], data => {
+    return api.query.assets.account.multi([[asset.typeExtras?.assetId, address]], data => {
```

Comment:
It will fast return in the next task regarding `Balance model`
```javascript
if (!assets.length || !api.query.assets) return Promise.resolve(noop);
```